packages/lesswrong/components/tagging/subforums/TagSubforumPage2.tsx

```diff
@@ -227,11 +226,17 @@ const TagSubforumPage2 = ({classes}: {
     skip: !tag
   })
 
-  const recordSubforumView = useRecordSubforumView({userId: currentUser?._id, tagId: tag?._id});
-  useEffect(() => {
-    if (!loadingTag && tag?._id)
-      void recordSubforumView();
-  }, [loadingTag, recordSubforumView, tag?._id]);
+  const { results: userTagRelResults } = useMulti({
+    terms: { view: "single", tagId: tag?._id, userId: currentUser?._id },
+    collectionName: "UserTagRels",
+    fragmentName: "UserTagRelDetails",
+    // Create a new UserTagRel if none exists. The check for the existence of tagId and userId is
+    // in principle redundant because of `skip`, but it would be bad to create a UserTagRel with
+    // a null tagId or userId so be extra careful.
+    createIfMissing: tag?._id && currentUser?._id ? { tagId: tag?._id, userId: currentUser?._id } : undefined,
```

Comment:
I feel a bit iffy about this (ex. does it try to validate the input?), but looks like it works and it's an optional arg so I'll give it a 🦭

Reply:
It calls the exact same function as the create mutation so it should do all the validations